include/seqan3/alphabet/gap/gap.hpp

```diff
@@ -31,117 +31,123 @@
 // DAMAGE.
 //
 // ==========================================================================
-// Author: Hannes Hauswedell <hannes.hauswedell@fu-berlin.de>
-// Author: Marcel Ehrhardt <marcel.ehrhardt@fu-berlin.de>
-// Author: David Heller <david.heller@fu-berlin.de>
-// ==========================================================================
+
+/*!\file
```

Comment:
Ok, then I agree, we should definitely do it this way. 
For consistency you can of course change it the current style and then change it back with another PR for the whole library¹, but from my POV it is ok to leave it like this, as well!

¹ That's what you proposed for the other changes :nail_care: